src/modules/PublicLab.TagsModule.js

```diff
@@ -16,7 +16,7 @@ module.exports = PublicLab.TagsModule = PublicLab.Module.extend({
     _module.key = 'tags';
     _module.options = options || _editor.options.tagsModule || {};
     _module.options.name         = 'tags';
-    _module.options.instructions = 'Tags relate your work to others\' posts. <a href="">Read more &raquo;</a>';
+    _module.options.instructions = 'Tags relate your work to others\' posts. <a href="/post" target="_blank">Read more &raquo;</a>';
```

Comment:
Could we change this URL to `https://publiclab.org/wiki/authoring-help#Tags` and we'll add that section in later?